modules/consul-demo-server/main.tf

```diff
@@ -8,11 +8,14 @@
  *
  */
 
+# Lookup the current AWS partition
+data "aws_partition" "current" {}
+
 module "consul-server" {
   source                  = "../single-node-asg"
   name_prefix             = "${var.name_prefix}"
   name_suffix             = "consul-server"
-  aws_cloud               = "${var.aws_cloud}"
+  aws_cloud               = "${data.aws_partition.current.partition}"
```

Comment:
If you drop this variable in the other modules, it won't be used here.

Reply:
So, just for clarity, what you're saying is that because no other modules should be using `aws_cloud` anymore, it's okay then to remove the `aws_cloud` variable that is declared here?